### ISSUE TITLE
Plink2 not reading bcf file

### ISSUE THREAD

**Issue by cmcouto-silva** [chrchang/plink-ng#119]
Trying to understand why `plink --bcf chr22.bcf --allow-extra-chr --make-bed --out chr22` (Plink version 1.9) works normally, but `plink2 --bcf chr22.bcf --allow-extra-chr --make-bed --out chr22` does not.

Error report:
```
./plink2 --bcf chr22.bcf --allow-extra-chr --make-bed --out chr22
PLINK v2.00a2LM 64-bit Intel (8 Jul 2019)      www.cog-genomics.org/plink/2.0/
(C) 2005-2019 Shaun Purcell, Christopher Chang   GNU General Public License v3
Logging to chr22.log.
Options in effect:
  --allow-extra-chr
  --bcf chr22.bcf
  --make-bed
  --out chr22
'
Start time: Wed Jul 10 22:32:40 2019
1483568 MiB RAM detected; reserving 741784 MiB for main workspace.
Using up to 160 threads (change this with --threads).
Error: Failed to open chr22-temporary.pgen.
End time: Wed Jul 10 22:32:40 2019
```

**Comment by chrchang** [comment]
This is because --bcf isn't implemented yet (it's now near the front of the to-be-implemented queue, though), and I had a typo in the code that was supposed to print the "--bcf is not implemented yet" error message.  (The typo is fixed by commit 651f393 .)

**Comment by cmcouto-silva** [comment]
Thank you, Christopher Chang!
Best,
